codalab/lib/worksheet_util.py

```diff
@@ -379,7 +379,8 @@ def render_dep(dep, show_key=True, show_uuid=False):
     elif genpath == 'summary':
         def friendly_render_dep(dep):
             key = dep['child_path'] or dep['parent_name']
-            value = key + '{' + (dep['parent_name'] + ':' if key != dep['parent_name'] else '') + \
+            friendly_parent_name = dep['parent_name'] or '<unnamed>'
```

Comment:
Use formatting.verbose_contents_str to be consistent with the way nulls are handled